editor/src/components/navigator/navigator-item/navigator-item-dnd-container.tsx

```diff
@@ -209,47 +224,61 @@ function onHoverDropTargetLine(
     : [EditorActions.setHighlightedView(propsOfDraggedItem.elementPath)]
 
   if (cursor == null || cursorDelta == null) {
-    return propsOfDraggedItem.editorDispatch(
-      [showNavigatorDropTargetHint(null, null, null)],
-      'leftpane',
-    )
+    return propsOfDraggedItem.editorDispatch([hideNavigatorDropTargetHint()], 'leftpane')
   }
 
-  const targetEntryWithReparentWiggle: { type: DropTargetType; entry: NavigatorEntry } | null =
-    (() => {
-      if (cursorDelta.x >= -BasePaddingUnit) {
-        return null
-      }
+  const targetEntryWithReparentWiggle: {
+    type: DropTargetType
+    targetParent: NavigatorEntry
+    indexPosition: IndexPosition
+  } | null = (() => {
+    const WiggleUnit = BasePaddingUnit * 1.5
```

Comment:
Very minor - we tend to only use PascalCase when a constant is global (or at least at the file level)

Reply:
ah OK, I'll move this constant to the file level

Reply:
https://github.com/concrete-utopia/utopia/pull/3716/commits/9943626c7ab22b69b23cf1d2a89e521b04ff203f